### PR TITLE
Fix #180790 www.merge.dev

### LINKED ISSUE #180790: www.merge.dev
### Issue URL (Annoyance)
 [https://www.merge.dev/blog/chatgpt-api-key](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.merge.dev%2Fblog%2Fchatgpt-api-key)

### Screenshots
<details>
  <summary>Screenshot 1 </summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard Browser Extension v4.3.53
System version: | Windows 11
Browser: | Edge
Stealth mode options: | Strip URLs from tracking parameters, <br>Hide your search queries, <br>Send Do-Not-Track header, <br>Self-destructing third-party cookies (2880), <br>Block trackers
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection, <br/>AdGuard URL Tracking<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances, <br/>AdGuard Cookie Notices, <br/>AdGuard Popups, <br/>AdGuard Mobile App Banners, <br/>AdGuard Other Annoyances, <br/>AdGuard Widgets, <br/>Adblock Warning Removal List

### PR DESCRIPTION
**Pull request by fix-parrot**
Fix #180790

### CODE REVIEW

File: AnnoyancesFilter/Cookies/sections/cookies_specific.txt [hunk 1]
```diff
@@ -276,6 +276,7 @@ consent-pref.trustarc.com#%#//scriptlet('trusted-click-element', 'div[aria-label
 !
 ! NOTE: Regular rules
 !
+merge.dev##.gdpr-cookie-banner
```
Comment (piquark6046) [inline AnnoyancesFilter/Cookies/sections/cookies_specific.txt:279]: Can be added into `surveymonkey.de,petri.com,surveymonkey.com,canadapost.ca,research.net,thurrott.com##.gdpr-cookie-banner`.

**Review by piquark6046 — CHANGES_REQUESTED** [review]
If you scroll down the website, you can see a popup about email subscription.

**Review by piquark6046 — CHANGES_REQUESTED** [review]
https://github.com/AdguardTeam/AdguardFilters/pull/180839#pullrequestreview-2099031887
Please resolve this.

**Review by zloyden — CHANGES_REQUESTED** [review]
@dakuzmin69 https://github.com/AdguardTeam/AdguardFilters/pull/180839#pullrequestreview-2099031887
<details><summary>Screenshot:</summary>

image

</details><br/>

**Review by zloyden — APPROVED** [review]

**Review by piquark6046 — APPROVED** [review]

**Pull request merged**